src/main/java/skinsrestorer/shared/storage/Locale.java

```diff
@@ -28,20 +28,20 @@
     public static String SKIN_CLEAR_SUCCESS = "&e[&2SkinsRestorer&e] &2Your skin has been cleared.";
     public static String SKIN_CLEAR_ISSUER = "&e[&2SkinsRestorer&e] &2Skin cleared for player %player.";
 
-    public static String MS_UPDATING_SKIN = "&e[&2SkinsRestorer&e] &2Uploading skin, please wait!";
+    public static String MS_UPDATING_SKIN = "&e[&2SkinsRestorer&e] &2Uploading skin, please wait...";
     public static String SUCCESS_UPDATING_SKIN = "&e[&2SkinsRestorer&e] &2Your skin has been updated.";
     public static String SUCCESS_UPDATING_SKIN_OTHER = "&e[&2SkinsRestorer&e] &2Skin updated for player %player.";
 
-    public static String ERROR_UPDATING_SKIN = "&e[&2SkinsRestorer&e] &4Error&8: &cAn error happened while updating your skin. Please try again later!";
-    public static String ERROR_MS_FULL = "&e[&2SkinsRestorer&e] &4MS Error&8: &cNo uploader accounts were available to fulfill your request. Try again later or consider https://mineskin.org/account";
+    public static String ERROR_UPDATING_SKIN = "&e[&2SkinsRestorer&e] &4Error&8: &cAn error occurred while updating your skin. Please try again later!";
+    public static String ERROR_MS_FULL = "&e[&2SkinsRestorer&e] &4MS Error&8: &cAPI timed out while uploading your skin. Please try again later. (mineskin)";
```

Comment:
```suggestion
    public static String ERROR_MS_FULL = "&e[&2SkinsRestorer&e] &4MS Error&8: &cAPI timed out while uploading your skin. Please try again later. (MineSkin)";
```

Maybe we should keep the link to the MineSkin accounts page